### ISSUE TITLE
Unexplained freeze with WINEDEBUG enabled

### ISSUE THREAD

**Issue by soredake** [lutris/lutris#797]

**Comment by strycore** [comment]
Why do you keep mentioning me on bug reports? also this has to little information for me to work on.

**Comment by soredake** [comment]
[lutris.log](https://github.com/lutris/lutris/files/1820267/lutris.log)
There is a log, one time it frozen at steam login screen, another time at 'launching game' screen.

**Comment by strycore** [comment]
The log shows nothing. You have process monitoring disabled, which you shouldn't have. Without the Wine output, there's nothing I can do.

**Comment by soredake** [comment]
![16-03-2018_20 07 09](https://user-images.githubusercontent.com/5204968/37537260-a5452dc0-2955-11e8-8191-e532e1703628.png)
Wine output is enabled, so, another bug?

**Comment by strycore** [comment]
No. You disabled the process monitor.

**Comment by soredake** [comment]
Get it, wine is freezing when debug info enabled and process monitor is enabled, is it normal?

**Comment by strycore** [comment]
Yes, it's perfectly normal, we have designed Lutris to freeze when those 2 options are enabled, for shits and giggles.

Seriously though. Wine doesn't freeze, there's something profoundly broken in your setup. Re-enable process monitor then repost the log so I can actually see what's going on.

**Comment by soredake** [comment]
After process monitor is enabled, no freeze.

**Comment by strycore** [comment]
Ok, so we're getting somewhere
